Packs/RiskIQDigitalFootprint/Integrations/RiskIQDigitalFootprint/RiskIQDigitalFootprint.yml

```diff
@@ -679,6 +695,14 @@ script:
       - 'false'
       required: false
       secret: false
+    - default: false
+      description: Digital Footprint (Global Inventory) assets potentially contain
+        pages of related data, for example attributes, cookies and host pairs. Size
+        determines the number for each of these associated items that are returned.
+      isArray: false
+      name: size
+      required: false
+      secret: false
```

Comment:
No `page` argument as well?

Reply:
@avidan-H It is an API limitation. RiskIQ's Get Asset API does not allow pagination through the data which is associated with the asset that is retrieved.